azure-kusto-data/tests/test_kusto_connection_string_builder.py

```diff
@@ -31,25 +31,41 @@ def test_aad_app(self):
         uuid = str(uuid4())
         key = "key of application"

```

Comment:
Maybe we want to a test that expects an error when kcsb is initialized without an authority id.